### PR TITLE
Standardisation of ConfigurationService

### PR DESCRIPTION
**Pull request by phil-pona**
- Default Values are now all defined on the ConfigurationService
- Default Values are shown on the ApplicationInfo Page
- AMW can now be configured via environment Variables as well (Priority: java System Properties, Env, default Value)

@yvespp review please

### CODE REVIEW

File: AMW_commons/src/main/java/ch/puzzle/itc/mobiliar/common/util/ConfigurationService.java [hunk 1]
```diff
@@ -117,7 +135,7 @@ public static String getProperty(ConfigKey key) {
 	 * @param defaultValue
 	 * @return
 	 */
-	public static Integer getPropertyAsInt(ConfigKey key, Integer defaultValue) {
+	protected static Integer getPropertyAsInt(ConfigKey key, Integer defaultValue) {
```
Comment (yvespp) [inline AMW_commons/src/main/java/ch/puzzle/itc/mobiliar/common/util/ConfigurationService.java]: Why protected?

File: AMW_commons/src/test/java/ch/puzzle/itc/mobiliar/common/util/ConfigurationServiceTest.java [hunk 2]
```diff
@@ -252,6 +252,16 @@ public void getKeyDefault() {
...
+
+	@Test
+	public void getDefaultValue() {
+		// when
+		String defaultValue = ConfigurationService.getDefaultValue(ConfigKey.LOCAL_ENV);
+
+		// then
+		assertEquals(ConfigKey.LOCAL_ENV.getDefaultValue(), defaultValue);
+	}
+
```
Comment (yvespp) [inline AMW_commons/src/test/java/ch/puzzle/itc/mobiliar/common/util/ConfigurationServiceTest.java]: Test for priority  java System Properties -> Env would be nice

**Comment by phil-pona** [comment]
@yvespp check out my update, the basic functionallity of overwriting config is now tested.

File: AMW_commons/src/main/java/ch/puzzle/itc/mobiliar/common/util/ConfigurationService.java (hunk 1 above)
Comment (yvespp) [inline AMW_commons/src/main/java/ch/puzzle/itc/mobiliar/common/util/ConfigurationService.java]: I mean, why isn't it public? All other methods that take a key and a defaultValue (e.g. `public static Boolean getPropertyAsBoolean(ConfigKey key, Boolean defaultValue)`) are public.

**Comment by yvespp** [comment]
LGTM thanks!
Please merge yourself once ready!

**Comment by phil-pona** [comment]
Thanks

**Pull request merged**